RetroBar/Themes/System.xaml

```diff
@@ -841,6 +845,39 @@
                    Name="ClockText">
         </TextBlock>
     </ControlTemplate>
+    
+    <Style TargetType="TextBlock"
+           x:Key="KeyboardLayout"
+           BasedOn="{StaticResource GlobalFonts}">
+        <Setter Property="TextOptions.TextRenderingMode"
+                Value="Auto" />
+        <Setter Property="Foreground"
+                Value="{DynamicResource KeyboardLayoutForeground}" />
+        <Setter Property="TextAlignment"
+                Value="Center" />
+        <Style.Triggers>
+            <DataTrigger Binding="{Binding Path=Orientation, RelativeSource={RelativeSource Mode=FindAncestor, AncestorType={x:Type Window}}}"
+                         Value="Vertical">
+                <DataTrigger.Setters>
+                    <Setter Property="Margin"
+                            Value="0,3" />
+                </DataTrigger.Setters>
+            </DataTrigger>
+        </Style.Triggers>
+    </Style>
+    
+    <ControlTemplate TargetType="{x:Type UserControl}"
+                     x:Key="KeyboardLayoutTemplateKey">
+        <Viewbox Width="24"
+                 MaxWidth="24"
+                 Stretch="Fill" StretchDirection="DownOnly"
+                 Margin="4,0,0,0">
+            <TextBlock Style="{DynamicResource KeyboardLayout}"
+                       Text="{Binding Path=LocaleIdentifierProperty, Converter={StaticResource cultureInfoToLocaleNameConverter}, ConverterParameter=TwoLetterIsoLanguageName, Mode=OneWay}"
+                       Name="KeyboardLayoutText">
+            </TextBlock>
+        </Viewbox>
```

Comment:
Done! 🙂 

Also, great point regarding the indicator appearance, I've initially made it from memory and it turned out that it didn't serve me that good in this particular case. 😆 The actual appearance was way better at anchoring user's view IMO so here it goes:

![image](https://github.com/dremin/RetroBar/assets/39274843/91c40191-991a-4d5d-a136-0811d2819c90)

It's not 100% historically accurate (for examaple, Win 95 used sentence case, e.g. "En" instead of "EN") but I think it looks miles better now. Side note, the padding of the text varies between themes in order to stay aligned with the clock.